CAT new form names

m-krysztofiak opened the pull request:
Copied the forms and removed 'cat' from the name - type is determined separately (adaptive or static)

m-krysztofiak committed:
cat new form names
copied the forms and removed 'cat' from the name

m-krysztofiak commented:
Tutaj jednak jeszcze są potrzebne poprawki, więc można się wstrzymać z recenzją :)

gkrajewski commented:
Proponuję przygotować całość na tym branchu i zrobić merge, jak już będzie wszystko gotowe

gkrajewski commented:
Teraz sobie uzmysłowiłem, że "całość" można różnie definiować 😅 Jest walidacja polska, norweska i dwujęzyczna, a w przypadku polskiej nie jest dla mnie jasne, czy chcemy się ograniczyć do WG, czy robić też jeszcze raz WS (osobiście jestem za tym drugim).

Nie mniej nie ma sensu robić merge, dopóki nie jesteśmy gotowi do puszczenia czegoś, jakiejś "całości".

m-krysztofiak committed:
instruction updates
zgodnie ze zmianami w mowadzieci

m-krysztofiak committed:
new cdi for cat validation
kopia pilotażu gram (do sonaty)

m-krysztofiak committed:
wg-a new form
copy of wg

m-krysztofiak committed:
ws-a updates
following mowadzieci text changes

m-krysztofiak committed:
wg-a form

m-krysztofiak committed:
settings for wg-a with no conditions

gkrajewski committed:
HTML in adaptive longText (per group)

m-krysztofiak committed:
update texts ws-a
zgodnie z mowa dzieci

m-krysztofiak committed:
wg-a text updated
zgodnie z mowa dzieci

m-krysztofiak committed:
cat wg new translations

m-krysztofiak committed:
items cat wg
dodane pytanie "Czy dziecko rozumie/mówi"

m-krysztofiak committed:
cat end message and minor correction

m-krysztofiak committed:
cat-wg minor improvements

m-krysztofiak committed:
cat-ws translations & engmessages

m-krysztofiak committed:
minor corrections

m-krysztofiak committed:
ws question with each item

m-krysztofiak committed:
ws minor corrections

m-krysztofiak committed:
wg minor changes

m-krysztofiak committed:
remove age limits for static cdi

m-krysztofiak committed:
name update
"-s" bc we first focus on validation in Poland and don't need forms with automatic redirections

m-krysztofiak committed:
cat end message update

m-krysztofiak committed:
new config files for CAT

m-krysztofiak committed:
end message update
uwzględniamy rodziców, którzy kończą badanie i są w trakcie

m-krysztofiak committed:
old cat forms removed

m-krysztofiak committed:
wg update se and max items

m-krysztofiak committed:
ws update se and max items

m-krysztofiak committed:
ws eng correct email

m-krysztofiak committed:
endmessage update

m-krysztofiak committed:
end message ws

gkrajewski committed:
Revert "old cat forms removed"

This reverts commit 19f0c55cb45760c806184a6cc79326c468817fc9.
